WNPRC_EHR/src/org/labkey/wnprc_ehr/pages/husbandry/WaterCalendar.jsp

```diff
@@ -814,7 +814,7 @@
                     $('#waterInformation').collapse('hide');
                     $('#waterTotalInformation').collapse('show');
                     WebUtils.VM.taskDetails["volume"](info.event.extendedProps.rawRowData.TotalWater.toString());
-                    WebUtils.VM.taskDetails["location"](info.event.extendedProps.rawRowData.location.toString());
+                    WebUtils.VM.taskDetails["location"](info.event.extendedProps.rawRowData["Id/curLocation/location"].toString());
```

Comment:
I'm not familiar with this particular function, but please make sure any values pulled directly from the db are html encoded if they are going to be used in the generated HTML from this JSP.